Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm

```diff
@@ -299,6 +299,33 @@ + (void)_deleteDefaultDataStoreForTesting
     return WebKit::WebsiteDataStore::deleteDefaultDataStoreForTesting();
 }
 
++ (void)_fetchAllIdentifiers:(void(^)(NSArray<NSUUID *> *))completionHandler
+{
+    auto completionHandlerCopy = makeBlockPtr(completionHandler);
+    WebKit::WebsiteDataStore::fetchAllDataStoreIdentifiers([completionHandlerCopy](auto&& identifiers) {
+        auto result = adoptNS([[NSMutableArray alloc] initWithCapacity:identifiers.size()]);
+        for (auto identifier : identifiers) {
+            if (auto uuid = identifier.createNSUUID())
+                [result addObject:uuid.get()];
+        }
+        completionHandlerCopy(result.autorelease());
+    });
+}
+
++ (void)_deleteDataStoreWithIdentifier:(NSUUID *)identifier completionHandler:(void(^)(NSError* error))completionHandler
```

Comment:
I would call this "_removeDataStoreWithIdentifier", since remove is the term we use for other data deletion in this API.

Reply:
Sure.